src/style.css

```
@@ -1,3 +1,84 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
 body {
+  --color: #00acdc;
+
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+header {
```

Use only class selector
```suggestion
.header {
```